demos/browser/server.js

```diff
@@ -49,7 +49,9 @@ if (captureS3Destination) {
 
 // return Chime Meetings SDK Client just for Echo Reduction for now.
 function getClientForMeeting(meeting) {
-  return ((useChimeSDKMeetings === 'true') || (meeting?.Meeting?.MeetingFeatures?.Audio?.EchoReduction === 'AVAILABLE')) ? chimeSDKMeetings : chime;
+  return (
+    (useChimeSDKMeetings === 'true') ||
+    (meeting && meeting.Meeting && meeting.Meeting.MeetingFeatures && meeting.Meeting.MeetingFeatures.Audio && meeting.Meeting.MeetingFeatures.Audio.EchoReduction === 'AVAILABLE')) ? chimeSDKMeetings : chime;
 }
```

Comment:
Could we update them to:
```js
function getClientForMeeting(meeting) {
  return useChimeSDKMeetings === "true" ||
    (meeting &&
      meeting.Meeting &&
      meeting.Meeting.MeetingFeatures &&
      meeting.Meeting.MeetingFeatures.Audio &&
      meeting.Meeting.MeetingFeatures.Audio.EchoReduction === "AVAILABLE")
    ? chimeSDKMeetings
    : chime;
}

```

Reply:
Prettified using online prettifier :)

Reply:
I sometimes use online prettier to format the code. Thanks!
https://prettier.io/playground/

Reply:
Used the same.